docs/book/user-guide/component-guide/orchestrators/vertex.md

```diff
@@ -50,24 +50,169 @@ To use the Vertex orchestrator, we need:
+    * permissions to create a Google Cloud Scheduler (e.g. with the [Cloud Scheduler Admin Role](https://cloud.google.com/iam/docs/understanding-roles#cloudscheduler.admin)).
+    * the [`Storage Object Creator Role`](https://cloud.google.com/iam/docs/understanding-roles#storage.objectCreator) to be able to write the pipeline JSON file to the artifact store directly (NOTE: not needed if the Artifact Store is configured with credentials or is linked to Service Connector)
+
+2. *the Vertex AI pipeline environment* is the GCP environment in which the pipeline steps themselves are running in GCP. The Vertex AI pipeline runs in the context of a GCP service account which we'll call here *the workload service account*.
+*The workload service account* can be explicitly configured in the orchestrator configuration via the `workload_service_account` parameter. If it is omitted, the orchestrator will use [the Compute Engine default service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account) for the GCP project in which the pipeline is running. This service account needs to have the following permissions:
+
+    * permissions to run a Vertex AI pipeline, (e.g. [the `Vertex AI Service Agent` role](https://cloud.google.com/vertex-ai/docs/general/access-control#aiplatform.serviceAgent)).
+
+3. *the scheduler Google Cloud Function* is a GCP resource that is used to trigger the pipeline on a schedule. This component is only needed if you intend on running Vertex AI pipelines on a schedule. The scheduler function runs in the context of a GCP service account which we'll call here *the function service account*. *The function service account* can be explicitly configured in the orchestrator configuration via the `function_service_account` parameter. If it is omitted, the orchestrator will use [the Compute Engine default service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account) for the GCP project in which the pipeline is running. This service account needs to have the following permissions:
+
+    * permissions to create a job in Vertex Pipelines, (e.g. [the `Vertex AI User` role](https://cloud.google.com/vertex-ai/docs/general/access-control#aiplatform.user)).
+
+4. *the Google Cloud Scheduler* is a GCP resource that is used to trigger the pipeline on a schedule. This component is only needed if you intend on running Vertex AI pipelines on a schedule. The scheduler needs a GCP service account to authenticate to *the scheduler Google Cloud Function*. Let's call this service account *the scheduler service account*. *The scheduler service account* can be explicitly configured in the orchestrator configuration via the `scheduler_service_account` parameter. If it is omitted, the orchestrator will use the following, in order of precedence:
```

Comment:
In points 3 and 4, I would explain their responsibilities a bit more in detail if possible. As far as I know, in the case of a scheduled pipeline, the cloud function takes over the responsibility of launching the vertex pipeline, whereas the scheduler makes sure that the function is executed on the given schedule.